src/components/Schedule/index.jsx

```diff
@@ -0,0 +1,15 @@
+import "./style.css";
+
+const Schedule = () => {
+    return(
+        <div id="schedule">
+            <h1 id="text">Schedule</h1>
+            <div id="calendar">
+                 <iframe title="calendar" src="https://calendar.google.com/calendar/embed?wkst=1&bgcolor=%23f3ecdc&ctz=America%2FLos_Angeles&mode=WEEK&src=Y19qcDlyM3J2dTJhOXRoYWE4am03a3Rodmdha0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23009688" 
```

Comment:
Nice work – schedule looks good! 👍 

One small nitpitck: Can we add like 20px of side margins on this calendar so it's not taking up the full width on smaller screens? Approving so you can merge once its fixed

![Screen Shot 2022-03-10 at 2 34 56 PM](https://user-images.githubusercontent.com/33165426/157766299-1022ffdd-f280-4752-8a02-621823f7983d.jpg)